addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -339,22 +356,31 @@ const Wysiwyg = Widget.extend({
         };
         const rpcMutex = new Mutex();
 
-        this._peerToPeerLoading = new Promise(async (resolve) => {
-            let iceServers = await this._rpc({route: '/web_editor/get_ice_servers'});
-            if (!iceServers.length) {
-                iceServers = [
-                    {
-                        urls: [
-                            'stun:stun1.l.google.com:19302',
-                            'stun:stun2.l.google.com:19302',
-                        ],
-                    }
-                ];
-            }
+        this._getCurrentRecord = async () => {
+            const records = await this._rpc({
+                model: modelName,
+                method: "read",
+                args: [
+                    [resId],
+                    [fieldName, 'write_date']
+                ],
+            });
+            records[0].body = records[0][fieldName];
+            return records[0];
+        }
+        this._getRecordWriteDate = (record) => {
+            const dateString = record.write_date.replace(/^(\d{4}-\d{2}-\d{2}) ((\d{2}:?){3})$/, '$1T$2Z');
+            return new Date(dateString);
+        }
 
-            this.ptp = new PeerToPeer({
-                peerConnectionConfig: { iceServers },
-                currentClientId: currentClientId,
+        this._getNewPtp = () => {
```

Comment:
It is not clear to me why those three functions are properties rather than actual methods ?

Reply:
1. because there is some closure in some two of them
2. that makes it clear it is only useful on collaborative mode